tests/include/sharedDummyHelpers.h

```diff
@@ -5,7 +5,7 @@
 
 #include <sys/file.h>
 
-enum MirrorRequest_Type { mirrorRequest_From = 1, mirrorRequest_To, mirrorRequest_Stop };
+enum MirrorRequestType { from = 1, to, stop };
```

Comment:
If you change names here, it would be better also to move to scoped enum:
enum class MirrorRequestType {...}; 
Only then, enum names from, to, stop are removed from global scope.